contracts/USM.sol

```diff
@@ -67,13 +70,27 @@ contract USM is IUSM, Oracle, ERC20WithOptOut, Delegable {
         buySellAdjustmentTimestamp: 0, buySellAdjustment: uint80(BILLION)       // initialize adjustment to 1.0 (scaled by 1b)
     });
 
-    constructor(Oracle oracle_, address[] memory optedOut_)
+    constructor(Oracle oracle0_, Oracle oracle1_, address[] memory optedOut_)
         ERC20WithOptOut("Minimalist USD v1.0", "USM", optedOut_)
     {
-        oracle = oracle_;
+        oracle0 = oracle0_;
+        oracle1 = oracle1_;
+        oracle = oracle0_;
         fum = new FUM(this, optedOut_);
     }
 
+    // ____________________ Admin-only external transactional functions ____________________
+
+    function setOracle(uint oracleIndex) external override {
```

Comment:
`onlyOwner()` modifier missing here?

Reply:
Yup, I left it out because I wasn't sure how best to implement it, multisig etc...  We'd def need to add `onlyOwner()` (or something) before merging.